iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/infospace/ImportInformationSpaceJobTest.java

```diff
@@ -0,0 +1,187 @@
+        String expectedOrganizationPath = Objects.requireNonNull(cl.getResource("eu/dnetlib/iis/wf/importer/infospace/output/organization.json")).getFile();
+        String expectedDocProjectPath = Objects.requireNonNull(cl.getResource("eu/dnetlib/iis/wf/importer/infospace/output/docproject.json")).getFile();
+        String expectedProjOrgPath = Objects.requireNonNull(cl.getResource("eu/dnetlib/iis/wf/importer/infospace/output/project_organization.json")).getFile();
+        String expectedDedupMappingPath = Objects.requireNonNull(cl.getResource("eu/dnetlib/iis/wf/importer/infospace/output/dedupmapping.json")).getFile();
+        String expectedReportPath = Objects.requireNonNull(cl.getResource("eu/dnetlib/iis/wf/importer/infospace/output/report.json")).getFile();
+        
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputDir.resolve(OUTPUT_NAME_DOCMETA).toString(), expectedDocumentPath, DocumentMetadata.class);
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputDir.resolve(OUTPUT_NAME_DATASET).toString(), expectedDatasetPath, DataSetReference.class);
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputDir.resolve(OUTPUT_NAME_PROJECT).toString(), expectedProjectPath, Project.class);
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputDir.resolve(OUTPUT_NAME_ORGANIZATION).toString(), expectedOrganizationPath, Organization.class);
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputDir.resolve(OUTPUT_NAME_DOC_PROJ).toString(), expectedDocProjectPath, DocumentToProject.class);
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputDir.resolve(OUTPUT_NAME_PROJ_ORG).toString(), expectedProjOrgPath, ProjectToOrganization.class);
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputDir.resolve(OUTPUT_NAME_DEDUP).toString(), expectedDedupMappingPath, IdentifierMapping.class);
+        
+        AvroAssertTestUtil.assertEqualsWithJsonIgnoreOrder(outputReportDir.toString(), expectedReportPath, ReportEntry.class);
+    }
+    
+    
+    private <T extends Oaf> void createGraphTableFor(String inputGraphTableJsonDumpPath,
+            String inputGraphTableDirRelativePath, Class<T> clazz, boolean isParquetFormat) {
+        Path inputGraphTableJsonDumpFile = Paths
+                .get(Objects.requireNonNull(cl.getResource(inputGraphTableJsonDumpPath)).getFile());
+
+        Dataset<T> inputGraphTableDS = readGraphTableFromJSON(inputGraphTableJsonDumpFile, clazz);
+        Path inputGraphTableDir = inputGraphDir.resolve(inputGraphTableDirRelativePath);
+        
+        if (isParquetFormat) {
+            inputGraphTableDS.write().format("parquet").save(inputGraphTableDir.toString());
+        } else {
+            inputGraphTableDS.toJSON().javaRDD().saveAsTextFile(inputGraphTableDir.toString());
+        }
+    }
```

Comment:
This implementation could be inverted and applied to reading graph tables, just as commented before :)